huaweicloud/resource_huaweicloud_cce_cluster_v3.go

```diff
@@ -130,6 +130,10 @@ func resourceCCEClusterV3() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validateIP,
 			},
+			"kube_proxy_mode": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
If the parameter can not be updated smoothly, please add the `ForceNew` behavior.